lib/msal-node/src/cache/serializer/Serializer.ts

```diff
@@ -84,7 +84,8 @@ export class Serializer {
                 key_id: atEntity.keyId,
                 token_type: atEntity.tokenType,
                 requestedClaims: atEntity.requestedClaims,
-                requestedClaimsHash: atEntity.requestedClaimsHash
+                requestedClaimsHash: atEntity.requestedClaimsHash,
+                userAssertionHash: atEntity.userAssertionHash
```

Comment:
Wasn't sure to follow camel or snake casing here (not sure why we switched). This method could possible be refactored to not require manual mapping from `AccessTokenEntity` to `SerializedAccessTokenEntity`